test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -64,6 +65,31 @@ public void LightGBMBinaryEstimator()
             Done();
         }
 
+        /// <summary>
+        /// LightGBMBinaryTrainer CorrectSigmoid test 
+        /// </summary>
+        [LightGBMFact]
+        public void LightGBMBinaryEstimatorCorrectSigmoid()
+        {
+            var (pipe, dataView) = GetBinaryClassificationPipeline();
+            var sigmoid = .789;
+
+            var trainer = ML.BinaryClassification.Trainers.LightGbm(new LightGbmBinaryTrainer.Options
+            {
+                NumberOfLeaves = 10,
+                NumberOfThreads = 1,
+                MinimumExampleCountPerLeaf = 2,
+                Sigmoid = sigmoid
+            });
+
+            var transformedDataView = pipe.Fit(dataView).Transform(dataView);
+            var model = trainer.Fit(transformedDataView, transformedDataView);
+
+            //the slope in the model calibrator should be equal to the negative of the sigmoid passed into the trainer
```

Comment:
nit: usually the format for comments is:
// Comment starting after a space with a capital letter and ending in a period.

We try to follow this format as often as possible.

Reply:
Sounds good. I will update it.

Reply:
Fixed.